app/src/selfRelease/java/org/radarcns/detail/UpdatesActivity.kt

```diff
@@ -130,47 +127,63 @@ class UpdatesActivity : AppCompatActivity(), TaskDelegate {
     }
 
     private fun checkForUpdates() {
-        val queue = Volley.newRequestQueue(this)
         val url = releasesUrl
```

Comment:
To reduce indentation, use
```suggestion
        val url = releasesUrl ?: return
```
then the check `url != null` is not needed.